homeassistant/components/esphome/icons.json

```diff
@@ -0,0 +1,20 @@
+{
+  "entity": {
+    "binary_sensor": {
+      "assist_in_progress": {
```

Comment:
I thought this entity was supposed to be removed in 2025.4.0?

Reply:
It is, but since it's still showing up, I went ahead and included it. We can always remove it later once the entity is officially gone. I wasn’t sure if the removal was delayed for some reason, so I figured it was safer not to leave it out just yet.

Reply:
It probably was forgotten to be removed